ci/Jenkinsfile.manual

```diff
@@ -0,0 +1,148 @@
+
+    parameters {
+        string(
+            name: 'PROJECT_REPO',
+            defaultValue: 'git@github.com:beau-lunarg/gfxreconstruct-pipeline-dev.git',
+            description: 'Project repository URL'
+        )
+        string(
+            name: 'PROJECT_BRANCH',
+            defaultValue: 'dev',
+            description: 'Project branch to build'
+        )
+        string(
+            name: 'TEST_REPO',
+            defaultValue: 'git@github.com:LunarG/VulkanTests',
+            description: 'Test repository URL'
+        )
+        string(
+            name: 'TEST_BRANCH',
+            defaultValue: '',
+            description: 'Test repository branch (leave empty for default)'
+        )
+        string(
+            name: 'TEST_SUITE_REPO',
+            defaultValue: 'git@github.com:LunarG/ci-gfxr-suites',
+            description: 'Test suite repository URL'
+        )
+        string(
+            name: 'TEST_SUITE_BRANCH',
+            defaultValue: '',
+            description: 'Test suite repository branch (leave empty for default)'
+        )
+        string(
+            name: 'TEST_SUITE',
+            defaultValue: 'commit-suite.json',
+            description: 'Test suite filename'
+        )
+        choice(
+            name: 'BUILD_MODE',
+            choices: ['Release', 'Debug'],
+            description: 'Build mode'
+        )
+        choice(
+            name: 'BITS',
+            choices: ['64', '32'],
+            description: 'Build architecture'
+        )
+        string(
+            name: 'NODES',
+            defaultValue: '',
+            description: 'Comma-separated list of nodes to run tests on'
+        )
+    }
+
+    options {
+        buildDiscarder(logRotator(
+            daysToKeepStr: '90',
+            numToKeepStr: '100',
+            artifactDaysToKeepStr: '90',
+            artifactNumToKeepStr: '100'
+        ))
+    }
+
+    stages {
+        stage('Run Tests') {
+            steps {
+                script {
+                    def tests = [:]
+
+                    node {
+                        checkout scm
+                        def runJob = load 'ci/runJob.groovy'
+
+                        def selectedNodes = params.NODES?.split(',')?.collect { it.trim() }?.findAll { it }
+
+                        if (!selectedNodes) {
+                            currentBuild.result = 'NOT_BUILT'
+                            echo 'No nodes selected. Run again with Build with Parameters.'
+                            return
+                        }
+
+                        selectedNodes.each { nodeName ->
+                            def type = runJob.getNodeType(nodeName)
+                            if (!type) {
+                                error "Cannot determine type for node: ${nodeName}. Name must contain 'and', 'ubu', 'mac', 'wnapl', or 'win'."
+                            }
+
+                            def closure
+                            switch (type) {
```

Comment:
This seems a little like an "anti-pipeline pattern," but this also might be the best way to "transition" from the current "UI configuration." Ideally, I would think each of these cases is a stage which only runs on a specific set of labels, which would hopefully remove the need for all of the `selectedNodes` and `runJob.getNodeType` logic.